src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -271,5 +271,24 @@ private ManifestVersion GetInstalledManifestVersion(ManifestId manifestId)
 
         internal static PackageId GetManifestPackageId(SdkFeatureBand featureBand, ManifestId manifestId) =>
             new PackageId($"{manifestId}.Manifest-{featureBand}");
+
+        // Note: cannot use Directory.Move because it errors when copying across mounts
+        internal static void CopyDirectory(string sourcePath, string destPath)
```

Comment:
Would it makes sense to make this an extension method  we could reuse in other places in the CLI if we ever needed something like this again?

Reply:
Makes sense, I'll make that change.